sdk/tables/Azure.Data.Tables/tests/TableEndpointType.cs

```diff
@@ -7,6 +7,7 @@ public enum TableEndpointType
     {
         Storage,
         CosmosTable,
-        StorageAAD
+        StorageAAD,
+        CosmosTableAAD
```

Comment:
Do we want to keep `AAD` or should we consider moving to the `Entra` branding?
_(not that it's important, because tests, but...)_

Reply:
It would re-record those tests if I renamed, so I'd prefer to keep it.